app/views/collection_drafts/forms/_descriptive_keywords.html.erb

```diff
@@ -34,6 +45,30 @@
                   <%= hidden_field_tag("draft[science_keywords][#{index}][variable_level_3]", science_keyword.fetch('VariableLevel3', '')) %>
                   <%= hidden_field_tag("draft[science_keywords][#{index}][detailed_variable]", science_keyword.fetch('DetailedVariable', '')) %>
                 </li>
+                <% selected_keywords_index += 1 %>
+              <% end %>
+
+              <% unless @keyword_recommendations.blank? %>
+                <% @keyword_recommendations.each_with_index do |recommended_keyword, recommendation_index| %>
+                  <% recommended_keyword_array = recommended_keyword.split('>') %>
```

Comment:
I think I would favor trimming the keywords so that we don't need to coordinate the change, then we can write a ticket to remove the trimming after they've changed.

Reply:
what do you mean by trimming, something like `strip`?

Reply:
I think I implemented this

Reply:
👍